source/package.json

```diff
@@ -27,10 +27,13 @@
     "jsii-pacmak": "^1.92.0",
     "tslint": "^5.20.1",
     "typescript": "4.7.4",
-    "aws-cdk-migration": "^1.135.0"
+    "aws-cdk-migration": "^1.135.0",
+    "constructs": "10.0.0"
```

Comment:
pinning constructs to `10.0.0` explicitly, without allowing for minor or patch upgrades seems intentional, so just curious about the reasoning?